### PR TITLE
Adding exceptions.yaml for 1.26

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

#### What type of PR is this:
/kind documentation

#### What this PR does / why we need it:

Adding `exceptions.yaml` file for v1.26 release

#### Special notes for your reviewer:

/assign @leonardpahlke

### CODE REVIEW

File: releases/release-1.26/exceptions.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  date_requested: 2022-10-10
+  date_reviewed: 2022-10-10
+  thread: "https://groups.google.com/g/kubernetes-sig-release/c/mSgvETUVYn0"
+  pull_requests:
+    - "https://github.com/kubernetes/kubernetes/pull/112643"
+  status: "approved"
+
+# Code Freeze Exceptions requested in 1.26
+
+codeFreeze:
```
Comment (leonardpahlke) [inline releases/release-1.26/exceptions.yaml:51]: We are missing one: https://groups.google.com/g/kubernetes-sig-release/c/pwb__uKXWjA/m/fFmG764gBAAJ

**Review by leonardpahlke — CHANGES_REQUESTED** [review]
we are missing one exception request

File: releases/release-1.26/exceptions.yaml (hunk 1 above)
Comment (marosset) [reply releases/release-1.26/exceptions.yaml:51]: Updated to add the StatefulSet Slice exception

**Review by leonardpahlke — APPROVED** [review]
LGTM

Thanks @marosset ! :)

**Comment by marosset** [comment]
/milestone v1.26

:)

**Pull request merged**